src/box/memtx_allocator.cc

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
Did you measure performance changes, just in case? We have `perf/tuple` test that checks tuple allocation RPS.